linode/manager
# test: [M3-8822] - Upgrade Cypress to v13.16.1

## Description 📝

This upgrades Cypress to v13.16.1 and `cypress-real-events` to v1.13.0. Check out the [Cypress changelog](https://docs.cypress.io/app/references/changelog) to see what's new since v13.11.x.

I normally wait for major releases or some compelling changes in a minor release before upgrading, but in this case I was checking to see if upgrading resolves the crash reporter issue we've discussed and it seems to fix it for me locally 🤞.

## Changes  🔄

- Upgrade Cypress to v13.16.1
- Upgrade cypress-real-events to v1.13.0

## Target release date 🗓️

N/A

## How to test 🧪

We can rely on the CI pipeline to make sure that this release doesn't cause any test breakage.

To see whether it resolves the crash reporter issue, you can follow these steps:

1. Check out PR, run `yarn && yarn build && yarn start:manager:ci`
2. In another terminal, run `yarn cy:run`. Let the tests run a moment then quit Cypress via `ctrl + c`
3. Confirm that after a few seconds no crash reporter dialog appears
4. Run `yarn cy:debug`, run a test, then quit Cypress via the CLI using `ctrl + c`
5. Confirm that after a few seconds no crash reporter dialog appears

## As an Author, to speed up the review process, I considered 🤔

👀 Doing a self review
❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
🤏 Splitting feature into small PRs
➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
🧪 Providing/improving test coverage
 🔐 Removing all sensitive information from the code and PR description
🚩 Using a feature flag to protect the release
👣 Providing comprehensive reproduction steps
📑 Providing or updating our documentation
🕛 Scheduling a pair reviewing session
📱 Providing mobile support
♿  Providing accessibility support


## As an Author, before moving this PR from Draft to Open, I confirmed ✅
